frontend/src/__tests__/snapshots/__snapshots__/App.test.tsx.snap

```diff
@@ -94,52 +94,6 @@ Array [
       <div
         className="chakra-stack css-84zodg"
       >
-        <div
```

Comment:
because of the conditional rendering, this component is being rendered by the snapshot and is therefore not being tested